### DISCUSSION TITLE
Improving the clothing system

### DISCUSSION THREAD

**Discussion by BROtania** [cataclysmbn/Cataclysm-BN#3377] [General]
I suggest, instead of assigning one encumbrance to all parts of the clothing body, make your own encumbrance index for each limb of the clothing.
This will make a more flexible balance and approach to dressing the character, and it's more logical.

**Comment by scarf005** [comment]
fixed by #2776. the docs are not there yet, but now it's possible to define separate coverage and encumbrance for individual bodypart.
